pkg/ibmcsidriver/controller.go

```diff
@@ -139,19 +245,28 @@ func (csiCS *CSIControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 		ctxLogger.Info("Volume Created", zap.Reflect("Volume", volumeObj))
 	}
 
+	// Prepare input for WaitForCreateVolumeAccessPoint
 	volumeAccesspointReq := provider.VolumeAccessPointRequest{
 		VolumeID: volumeObj.VolumeID,
-		VPCID:    os.Getenv("VPC_ID"),
 	}
 
-	//Create VolumeAccess Point
-	//No need to check for access point existence as library takes care of the same
-	volumeAccessPointObj, err := createVolumeAccessPoint(session, volumeAccesspointReq, ctxLogger)
+	volumeAccessPoints := volumeObj.VolumeAccessPoints
```

Comment:
As of now there is no functional impact of this condition. Even if do this check it will go ahead and just check volumeAccessPoint status is pending or stable. In all cases if CreateVolume is success it is 100% guaranteed that VolumeAccessPoint is created otherwise CreateVolume as whole will fail. So there should be always volumeAccessPoint for our PVC volume. This will go ahead and check for status of access point and take action and k8s will retry on top of library retries if accesspoint status is not valid

As discussed we will shift all this logic to library later.